### PR TITLE
Trad/tools

### PR DESCRIPTION
**Pull request by fdechelle**
Translated modules/tools.

### CODE REVIEW

File: source/modules/tools/index.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Tools module
+============
+
+Tools module allows to manage projects, reminders, RSS feeds, knowledge base, reservations and to generate reports.
```
Comment (cconard96) [inline source/modules/tools/index.rst:4]:
```suggestion
The GLPI Tools module is used to manage projects, reminders, RSS feeds, knowledge base, reservations and to generate reports.
```
This language matches the other translated modules

File: source/modules/tools/projects.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+.. image:: images/task-details-project.png
+   :alt: A task
+   :align: center
+
+.. note::
+   A project task is not an independent item, it can only be accessed through the `Project tasks` tab of the project owning the task.
+
+Tasks can be attached to tickets in order to plan interventions.
+
+Provisional and effective duration are also defined for a task, reported at project level together with attached tickets duration.
```
Comment (cconard96) [inline source/modules/tools/projects.rst:127]:
```suggestion
Planned and actual duration are also defined for a task, reported at project level together with attached tickets duration.
```

File: source/modules/tools/reminders.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
+Manage personal or public reminders
+===================================
+
+A reminder in GLPI is a simple text with attached files;; it allows to transmit information to other users of the platform.
```
Comment (cconard96) [inline source/modules/tools/reminders.rst:4]:
```suggestion
A reminder in GLPI is simple text with attached files; it allows to transmit information to other users of the platform.
```

File: source/modules/tools/rssfeed.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+   :align: center
+
+
+The different tabs
+------------------
+
+Content
+~~~~~~~
+
+This tab allows to consul the number of displayed items; this content is the one displayed on home page.
```
Comment (cconard96) [inline source/modules/tools/rssfeed.rst:24]:
```suggestion
This tab allows to consult the number of displayed items; this content is the one displayed on home page.
```

**Review by cconard96 — CHANGES_REQUESTED** [review]

**Review by cconard96 — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Pull request merged**